### ISSUE TITLE
Python build tools should use Python 3

### ISSUE THREAD

**Issue by benjaoming** [learningequality/kolibri-installer-windows#163]
The helper scripts in `src/python-scripts` should not require Python 2.7 (README indicates this) - hopefully this is just a matter of trying them out on Python 3 and updating README.

**Comment by cpauya** [comment]
I agree, we need to test this out now that [Python 2.7.18 is the last release for Python 2](https://pythoninsider.blogspot.com/2020/04/python-2718-last-release-of-python-2.html).

/cc: @mrpau-richard
